components/admin/requests/processing/TasksCard.tsx

```diff
@@ -201,9 +203,40 @@ export default function ProcessingTasksCard({ applicationId }: ProcessingTasksCa
             </Button>
           )}
         </ProcessingTaskStep>
-        {/* Task 4: Assign invoice number: Assign number (MODAL) */}
+        {/* Task 4: Review Request Information -> Disable Editing of First 3 steps */}
         <ProcessingTaskStep
           id={4}
+          label={`Review request information`}
+          description="Editing will be disabled upon completion of this step"
+          isCompleted={reviewedRequestInformation}
+        >
+          <ReviewInformationModal
+            applicationId={applicationId}
+            requestType={'Request'}
+            onConfirmed={() => setReviewedRequest(true)}
+          >
+            {invoiceNumber === null ? (
+              <Button
+                marginLeft="auto"
+                height="35px"
+                bg="background.gray"
+                _hover={{ bg: 'background.grayHover' }}
+                color="black"
+              >
+                <Text textStyle="xsmall-medium">Review Information</Text>
+              </Button>
+            ) : (
+              <Button variant="ghost" textDecoration="underline black">
```

Comment:
Does `underline black` work here? If we want black text we specify that in the `color` prop